servers/zelicraft/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "zelicraft network",
+    "name": "Zelicraft Network",
+    "description": "Servidor de habla hispana, servidor de PvE y PvP. ¡Únete hoy!",
+    "addresses": [
+        "zelicraft.com",
+        "dev.zelicraft.com",
+        "potyh.es",
+        "followersland.me"
+    ],
+    "primaryAddress": "zelicraft.com",
+    "minecraftVersions": [
+        "1.19.*"
+    ],
+    "primaryMinecraftVersion": "1.19.3",
+    "primaryColor": "#8E00DD",
+    "secondaryColor": "#FDD1FF",
+    "primaryLanguage": "es-EN",
+    "languages": [
+        "es-EN"
+    ],
+    "primaryRegion": "NA",
+    "gameTypes": [
+        "PvP",
+        "PvE",
```

Comment:
the max game types is 3 please remove some

Reply:
Ready, you should be all set.